tables/checks.py

```diff
@@ -49,39 +71,57 @@ def is_float(element):
 
 
 @pa.extensions.register_check_method(check_type=CheckType.ELEMENT_WISE)
-def not_in_future(element):
+def not_in_future(element: Any):
     """Checks that a datetime is not in the future.
 
     :param element: an element to check
     :return: True if passes the check, else False
     """
+    if _should_fail_check_because_element_is_nan(element):
+        return False
+
+    # If it's not come through here as a string, it's probably some other native data type like an int or string,
```

Comment:
"some other native data type like an int or **string**"

is this a typo?

Reply:
oops yeah - we expect it here as a datetime - good spot

Reply:
fixed